js/functions.js

```diff
@@ -51,3 +51,21 @@ const getNumber = (string) => {
 };
 
 getNumber('2023 год');
+
+const getTime = (startWork, endWork, startMeeting, meetingDuration) => {
+  const dayStartMinutes = timeToMinutes(startWork);
+  const dayEndMinutes = timeToMinutes(endWork);
+  const meetingStartMinutes = timeToMinutes(startMeeting);
+  const meetingEndMinutes = meetingStartMinutes + meetingDuration;
+
+  return meetingStartMinutes >= dayStartMinutes && meetingEndMinutes <= dayEndMinutes;
+};
+
+function timeToMinutes(time) {
+  const parts = time.split(':').map(Number);
```

Comment:
Можно сразу достать через деструктуризацию
```suggestion
  const [hours, minutes] = time.split(':').map(Number);
```